src/CheckOut.js

```diff
@@ -16,7 +16,7 @@ import {
   Paneset,
   Button,
 } from '@folio/stripes/components';
-import { Pluggable } from '@folio/stripes/core';
+import { IfPermission, Pluggable } from '@folio/stripes/core';
```

Comment:
Typically in a project, we try to follow to code style that if more than one property is imported, each of them should be placed on a separate line.
```suggestion
import {
  IfPermission,
  Pluggable,
} from '@folio/stripes/core';
```